docs/css/style.scss

```diff
@@ -85,18 +85,217 @@ nav {
 }
 
 .main {
+    font-family: 'Karla', Helvetica;
 
     h1 {
-        font-size: 2.625rem;
-        font-weight: 600;
-        line-height: 3rem;
+        font-family: 'Karla', Helvetica;
+        font-size: 38px;
+        font-weight: bold;
+        line-height: 44px;
```

Comment:
[non-blocking] I'd recommend using `rem` instead of `px` for line-height and font-size. Here's some more info: https://css-tricks.com/is-it-better-to-use-ems-rems-than-px-for-font-size/